user-onboarding/src/components/users.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export default function Users({ details }) {
+   if (!details) {
+      return (
+         <h3>Fetching Users...</h3>
+      )
+   }
+
+   const detailsArr = new Array(details)
+
+   return (
+      <div className='user container'>
+         <h2>{details.first_name} {details.last_name}</h2>
+         <p>Email: {details.email}</p>
+         <p>Role: {details.role}</p>
+         <p>Agreed to Terms?
+            <span className='userTerms'>{detailsArr.map(item => {
```

Comment:
You don't need to map through the array here. You can just get the necessary information from the detail object and render based on that condition.

example
{details.item === true
                  &&
                  <p>yes</p>
               }

would render yes if the item key in the details array is true.